programs/Makefile

```diff
@@ -57,7 +57,13 @@ endif
 ifneq (,$(filter Windows%,$(OS)))
 EXT =.exe
 VOID = nul
-RES_FILE = ..\projects\VS2010\zstd\zstd.res
+RES64_FILE = ..\projects\VS2010\zstd\generate_res\zstd64.res
```

Comment:
@inikep : 
I did not realize that with this change, `programs` is now dependent on `projects` (recently renamed `build`).
which does not feel correct : `programs` should not depend on `build`.

So, would it be possible to have `zstd??.res` stored within `programs` instead ?
I guess it would change the corresponding Visual projects ?